saagieapi/jobs/jobs.py

```diff
@@ -892,3 +892,83 @@ def export(
             logging.warning("❌ Job [%s] has not been successfully exported", job_id)
             write_error(error_folder, "jobs", job_id)
         return result
+
+    def import_job(
+        self,
+        job_info: str,
+        project_id: str,
+        path_to_package: str = None,
+    ) -> bool:
+        """Import a job from a folder
+
+        Parameters
+        ----------
+        job_info : str
+            Info of the job in JSON format
+        project_id : str
+            Project ID to import the job
+        path_to_package : str, optional
+            Path to the package of the job to import
+        Returns
+        -------
+        bool
+            True if job is imported False otherwise
+        """
+        result = True
+
+        try:
+            jobId = job_info["id"]
```

Comment:
all variables should be in snake_case and not camel_case

Reply:
changes done